### DISCUSSION TITLE
Ability to send (discussion) feedback via button in settings

### DISCUSSION THREAD

**Discussion by Joehuu** [ppy/osu#22735] [Ideas] [answered]
Since *new users* can download lazer on the [official download page](https://osu.ppy.sh/home/download), they may not have knowledge of where to send feedback (if they want to) since they never visited GitHub.

There will be a "send feedback" button that opens a popup saying "we use github discussions and requires creating an account there. do you want to continue?" or similar and saying yes opens the link here.

Ignore this idea if we don't want to have an influx of potentially low-effort discussions/issues, but that time will come eventually (when lazer *releases* so why not now?).

**Comment by mk56-spn** [comment]
This does sound cool but as you say the potential for an explosion of pointless posts is a real issue and yeah it will probably happen on release but delaying the inevitable is still better imo since it gives Devs and contributors more time to comb over known issues that will likely come up a dozen times again if we were to implement this. Definitely a cool idea just not sure it would overall be good

**Comment by bdach** [comment, marked as answer]
Without any method to prevent submission of duplicates (and what github currently provides is *woefully* inadequate) doing this would probably make me stop looking at discussions. The signal-to-noise ratio will be too high for discussions to be useful again if you do this, at least with the amount of moderation/triage we have available right now.

**Reply by peppy** [reply]
Pretty much this. Sentry already has the ability to harvest user feedback but I've not bothered due to the reasons cited here.

At some point we'll probably add a feedback form for when an error is detected (just to get more insight from the user to attach to error reports) but that's about as far as I'd see it going, short term anyway.
